src/osd/osd_types.cc

```diff
@@ -3125,6 +3127,9 @@ void pg_stat_t::decode(ceph::buffer::list::const_iterator &bl)
       decode(objects_trimmed, bl);
       decode(snaptrim_duration, bl);
     }
+    if (struct_v >= 29) {
+      decode(log_dups_size, bl);
+    }
```

Comment:
What otherwise? It doesn't seem we initialize `log_dups_size` anyhow, even in ctor.

Reply:
Right, added it to the ctor, it will be zero in case of decode is less then 29